src/mtev_http1.c

```diff
@@ -996,6 +1000,15 @@ mtev_http1_session_req_consume_read(mtev_http1_session_ctx *ctx,
       }
     }
     else {
+      /* Check if we have enough space in this bchain node for the chunk header "$chunk_size\r\n" */
+      if(head->allocd - head->start < MAX_CHUNK_HEADER_SIZE) {
```

Comment:
I think you still need the old code if this predicate fails.  And this is <= no?

Reply:
Regarding `<=`
- If we have equality then, the header should fit and we are good.
- In any case, MAX_CHUNK_HEADER_SIZE == 20, is just a guess, so `<` vs. `<=` is not an important edge case to consider.

If this condition fails, the current bchain node is large enough for our needs. The subsequent reads will be able to get the chunk header field into the buffer.
Once we have this, the section `if(str_in_f) { ... }` will prepare for the chunk payload.